vm-virtio/src/vsock/packet.rs

```diff
@@ -96,6 +101,31 @@ pub struct VsockPacket {
     buf_size: usize,
 }
 
+fn get_region_host_address_range(
```

Comment:
The real question is where to move it. The best place would be `vm-memory`, but since we cannot, it's kind of a tricky one. We can put it into `vm-virtio/src/lib.rs` but that means other piece of code like VFIO would need to import virtio for this.

Reply:
There were more places where `get_host_address` is called, I moved our wrapper to the `vm-device` crate.